app/src/main/res/layout/content_list_fragment.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools">
+  <data>
+    <variable
+      name="contentFragment"
+      type="org.oppia.app.player.content.ContentListFragment"/>
+    <variable
```

Comment:
Are either of these two variables used? If not, please remove.

Reply:
Its not used so removed.